src/summary/LongitudinalTableVisualizer.jsx

```diff
@@ -91,12 +75,12 @@ export default class LongitudinalTableVisualizer extends Visualizer {
         return (
             <div>
                 <LongitudinalTable
-                    reorderRows={this.reorderRows}
-                    dataInfo={this.state.data}
+                    preferenceManager={this.props.preferenceManager}
+                    reorderRows={this.reorderRows} dataInfo={this.state.data}
                     tdpSearchSuggestions={this.props.tdpSearchSuggestions}
                     conditionSectionName={this.props.conditionSection.name}
                     subsectionLabel={this.props.conditionSection.subsectionLabel}
-                    preferenceManager={this.props.preferenceManager} />
+                    pluralLabel={this.props.conditionSection.name.toLowerCase()} />
```

Comment:
Minor comment: could we call this pluralSectionLabel? Not clear from the prop name to me what exactly is being labelled.